infrastructure/src/main/resources/logback.prod.xml

```diff
@@ -27,6 +27,7 @@
     <appender name="async_file" class="ch.qos.logback.classic.AsyncAppender">
         <appender-ref ref="file" />
         <queueSize>1024</queueSize>
+        <maxFlushTime>5000</maxFlushTime>
```

Comment:
Should we remove this line: https://github.com/CorfuDB/CorfuDB/blob/master/infrastructure/src/main/resources/logback.prod.xml#L5 , since we are doing it elsewhere and it's missing class name as well?

Reply:
Yep, I think we should keep maxFlushTime